test/built-ins/Temporal/PlainDateTime/prototype/since/weeks-months-mutually-exclusive.js

```diff
@@ -0,0 +1,24 @@
+// Copyright (C) 2022 Igalia, S.L. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: sec-temporal.plaindatetime.prototype.since
+description: Weeks and months are mutually exclusive
+includes: [temporalHelpers.js]
+features: [Temporal]
+---*/
+
+const dt = new Temporal.PlainDateTime(1976, 11, 18, 15, 23, 30, 123, 456, 789);
+const laterDateTime = dt.add({ days: 42, hours: 3 });
+
+TemporalHelpers.assertDuration(
+  laterDateTime.since(dt, { largestUnit: "weeks" }),
+  0, 0, 6, 0, 3, 0, 0, 0, 0, 0,
+  "weeks and months are mutually exclusive (1)"
```

Comment:
For this message I'd recommend replacing "(1)" and "(2)" with something like "(prefer weeks)" and "(prefer months)"

Reply:
done